docs/BUILD.md

```diff
@@ -298,30 +300,30 @@ The following custom packages are used:
 
 * kernel: EVE uses its own custom kernel package, rather than one of the standard linuxkit ones. This is primarily due to kernel modules and drivers, especially on arm, as well as Xen requirements.
 * `init` packages:
-    * `zededa/grub` - 
-    * `zededa/devices-trees` - 
-    * `zededa/fw` - 
-    * `zededa/xen` - 
-    * `zededa/gpt-tools` - 
-    * `zededa/dom0-ztools` - 
+    * `zededa/grub` - CoreOS inspired GRUB required to enable CoreOS-style dual partition upgrades.
```

Comment:
This comment is good enough for here, but worth going into greater detail in the grub list of patches further below as to what is added. Is it something binary?

Reply:
Excuse me, I meant in the grub list of patches in [UPSTREAMING.md](./UPSTREAMING.md)